apps/dashboard/app/helpers/batch_connect/sessions_helper.rb

```diff
@@ -195,17 +195,37 @@ def status_context(session)
     end
   end
 
+
+  def terminate_delete(session)
+    if Configuration.terminate_session_enabled && !session.completed?
+      terminate(session)
+    else
+      delete(session)
+    end
+  end
+
   def delete(session)
     link_to(
       fa_icon('trash-alt', classes: nil) << ' ' << t('dashboard.batch_connect_sessions_delete_title'),
-      session,
+      batch_connect_session_path(session.id),
       method: :delete,
       class: "btn btn-danger float-right btn-delete",
       title: t('dashboard.batch_connect_sessions_delete_hover'),
       data: { confirm: t('dashboard.batch_connect_sessions_delete_confirm'), toggle: "tooltip", placement: "bottom"}
     )
   end
 
+  def terminate(session)
+    link_to(
+      fa_icon('trash-alt', classes: nil) << ' ' << t('dashboard.batch_connect_sessions_terminate_title'),
```

Comment:
The icon should likely be different. Maybe `ban-alt`? I'm not sure after searching FA site for a bit. (there are no stop signs, and stop & play make more sense for music apps and so on)

Reply:
It looks like `ban-alt` is not in Awesome 5 Free, which is the font family we are using.
These are some alternative:

<img width="445" alt="Screenshot 2022-12-15 at 09 02 55" src="https://user-images.githubusercontent.com/17336210/207823125-844d6161-778c-43dd-b96c-2a5c6e4bb255.png">

<img width="394" alt="Screenshot 2022-12-15 at 09 04 43" src="https://user-images.githubusercontent.com/17336210/207823127-1eddc620-c829-4b21-b428-87dc49196b21.png">

<img width="418" alt="Screenshot 2022-12-15 at 09 00 27" src="https://user-images.githubusercontent.com/17336210/207823116-f9f79a91-1f2b-4ea9-bfcc-bd3454ad385f.png">

<img width="429" alt="Screenshot 2022-12-15 at 09 00 44" src="https://user-images.githubusercontent.com/17336210/207823121-dea39618-f896-439e-bf4a-9fbceebf01d0.png">

Reply:
I'm thinking the circle with the X?  I'm also wondering if `Terminate` is the right word. Maybe just `Stop`?  (externally, I'm not concerned with the route or function names)

Parlance is all over the place with every scheduler saying something different. I have no idea what someone may infer from Terminate, but it seems to me that we need to limit what someone could infer or assume about it.
https://slurm.schedmd.com/rosetta.pdf

Reply:
We went through the same process when we implemented this feature internally.
We thought that `stop` might imply that the job could be `started` later. But it was an assumption.

Here is the button with Stop for reference:
<img width="373" alt="Screenshot 2022-12-15 at 15 59 25" src="https://user-images.githubusercontent.com/17336210/207908497-e5399459-85a7-47f8-873d-de258903ac09.png">

Reply:
Another idea from the rosetta, what about `cancel`?

<img width="397" alt="Screenshot 2022-12-15 at 16 07 47" src="https://user-images.githubusercontent.com/17336210/207910202-b2f31084-de5e-4c3e-af0d-99184a8f91c4.png">

Reply:
> We thought that `stop` might imply that the job could be `started` later. But it was an assumption.

At the time, this was in the context of a "Stop" button with the music-player stop icon.

I think that the "x in a circle" icon makes it more clear that stopping the job is an irreversible action.

Reply:
What is your preference @johrstrom?
To make the updates to the text and messages.

Reply:
`cancel or `stop` seems fine. I linked the Rosseta stone for schedulers - so it's not like there's 1 term everyone uses.

I think either one is going to hit some folks and confuse others, so it's really a toss up for me.

Reply:
I am leaning towards `cancel`. Updating button and messages accordingly.

Reply:
That'll work just fine for me.

Reply:
Code updated to `cancel`